dev-libs/weston/metadata.xml

```diff
@@ -11,7 +11,7 @@
 	<flag name="editor">Install wayland-editor example application</flag>
 	<flag name="fbdev">Enable fbdev compositor support</flag>
 	<flag name="fullscreen">Enable fullscreen/kiosk shell</flag>
-	<flag name="gles2">Use GLESv2 cairo instead of full GL</flag>
+	<flag name="gles2-only">Use GLESv2 cairo instead of full GL</flag>
```

Comment:
This description has actually been wrong for a while. For the time being at least, it has nothing to do with Cairo and simply enables the GLES2 renderer. Full OpenGL is never used by Weston itself.

Reply:
I guess in the end using `gles2support` here might be a good idea because I wouldn't be surprised that it would be the same situation for stuff like Qt.

Reply:
I don't know what you mean. Just `gles2` works for me but with a better description.

Reply:
Well, `gles2support` would allow to easily know if it's something from when this flag was a mess, but yeah description is probably enough.